src/components/menuBar/menu-bar.spec.js

```diff
@@ -26,11 +26,27 @@ describe('material.components.menuBar', function() {
       });
 
       describe('ARIA', function() {
+        
         it('sets role="menubar" on the menubar', function() {
           var menuBar = setup();
           var ariaRole = menuBar[0].getAttribute('role');
           expect(ariaRole).toBe('menubar');
         });
+        
+        it('should set the role on the menu trigger correctly', inject(function($compile, $rootScope) {
```

Comment:
@topherfangio  Yeah, the error will be thrown. 

About the spaces inside of the string - I agree, this will work better with code formatting. But someone of the Team told me, that this is the better way to do it.

Actually, this is really inconsistent in the whole project. Some specs are doing it the current way, and some doing it your way. Not sure, what's right to go with :smile: 
I think it should be fine.